extensions-contrib/kafka-emitter/pom.xml

```diff
@@ -117,5 +117,46 @@
       <version>${project.parent.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>${protobuf.version}</version>
+    </dependency>
+      <dependency>
+          <groupId>com.google.protobuf</groupId>
+          <artifactId>protobuf-java-util</artifactId>
+          <version>${protobuf.version}</version>
```

Comment:
the version is already defined by the dependencyManagement section in the parent pom, so no need to specify it here.